scripts/tests/test_scripts.py

```diff
@@ -330,8 +330,9 @@ def test_export(task):
 @pytest.mark.integration
 @pytest.mark.parametrize('sentencepiece', [False, True])
 def test_finetune_squad(sentencepiece):
-    arguments = ['--optimizer', 'adam', '--batch_size', '12',
-                 '--gpu', '0', '--epochs', '2', '--debug']
+    arguments = ['--optimizer', 'adam', '--batch_size', '32',
```

Comment:
@zburning seems this test not runs 1195.36s and is the longest running test. Would it be reasonable to reduce runtime?